book-crawler/tests/test_async_api.py

```
@@ -0,0 +1,81 @@
+    # 1. 크롤링 시작
+    print("\n1. 크롤링 작업 시작...")
+    response = requests.post(f"{API_URL}/crawl/kyobo/async", params={"max_pages": 1})
+    
+    if response.status_code != 200:
+        print(f"❌ 에러: {response.status_code} - {response.text}")
+        return
+        
+    data = response.json()
+    job_id = data["job_id"]
+    
+    print(f"✅ 작업 시작됨!")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)